src/Compilers/VisualBasic/Portable/Symbols/WellKnownMembers.vb

```diff
@@ -653,6 +653,75 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return MyBase.MatchTypeToTypeId(type, typeId)
             End Function
         End Class
+
+        Friend Function SynthesizeTupleNamesAttributeOpt(type As TypeSymbol) As SynthesizedAttributeData
+            Debug.Assert(type IsNot Nothing)
+            Debug.Assert(type.ContainsTuple())
+
+            Dim stringType = GetSpecialType(SpecialType.System_String)
+            Debug.Assert(stringType IsNot Nothing)
+            Dim names As ImmutableArray(Of TypedConstant) = TupleNamesEncoder.Encode(type, stringType)
+
+            ' If there are no names, elide the attribute entirely
```

Comment:
made it an assert

---
In reply to: [79721487](https://github.com/dotnet/roslyn/pull/13923#discussion_r79721487) [](ancestors = 79721487)